assignments/prototypes.js

```diff
@@ -23,6 +32,20 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+
+
+function CharacterStats(attributes){
+  GameObject.call(this, attributes);
+  this.hp = attributes.hp;
+  this.name = attributes.name;
```

Comment:
You _could_ use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here so as to not have to repeat `attributes` over again and again:
```javascript
function CharacterStats({ hp, name }){
  GameObject.call(this, { hp, name });
  this.hp = hp;
  this.name = name;
}
```